api/v1alpha1/imagelist_types.go

```diff
@@ -31,6 +31,8 @@ type ImageListStatus struct {
 	Success int64 `json:"success"`
 	// Number of nodes that failed to run the job
 	Failed int64 `json:"failed"`
+	// Number of nodes that were skipped, e.g. because they are a non-linux node
```

Comment:
let's keep it generic (say "due to skip selector" or something), we can specific non-Linux in docs

Reply:
👍 

- [ ] Fix comment